File: src/test/resources/output/java/unirest/full.java
HttpResponse<String> response = Unirest.post("http://mockbin.com/har?baz=abc&foo=bar&foo=baz&key=value")
  .header("Cookie", "foo=bar; bar=baz")
  .header("content-type", "application/x-www-form-urlencoded")
  .header("accept", "application/json")
  .field("foo","bar")
  .asString();

File: src/test/resources/output/ruby/nethttp/full.rb
require 'uri'
require 'net/http'

url = URI("http://mockbin.com/har?baz=abc&foo=bar&foo=baz&key=value")

http = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Post.new(url)
request["Cookie"] = "foo=bar; bar=baz"
request["content-type"] = "application/x-www-form-urlencoded"
request["accept"] = "application/json"
request.body = "foo=bar"

response = http.request(request)
puts response.read_body
